### ISSUE TITLE
Analyze MEF exports for correctness

### ISSUE THREAD

**Issue by dgriffen** [Microsoft/VSSDK-Analyzers#43]
There should be a diagnostic checking if `Export` declarations are correct. For example:

```csharp
[Export(typeof(IBar))]
public class Foo { }
```

This should be flagged as incorrect since `Foo` does not implement `IBar`
